tasks_hayahammad.md

```diff
@@ -0,0 +1,104 @@
+# First step on command line.
+1. Display your current directory.
+#pwd 
+2. Change to the /etc directory.
+#cd /etc
+3. Now change to your home directory using only three key presses.
+#cd ~ 
+4. Go to the parent directory of the current directory.
+#cd ..
+5. Go to the root directory.
+#cd /
+6. List a long listing of the root directory using two ways.
+1.#ls -l 
+2.#ls -la 
+7. Using man pages list all files under /usr/bin and show list of files in long listing format,human readable size format,and show the oldest files first.
+#ls /usr/bin
+#ls -lh  /usr/bin
+#ls -lrt  /usr/bin
+8. Stay where you are, and list the contents of /bin and /sbin.
+#ls /bin
+#ls /sbin
+9. List the files in /boot in a human readable format.
+#ls -lh /boot
+10. Create in one command the directories ~/dir1/dir2/dir3 (dir3 is a subdirectory
+from dir2, and dir2 is a subdirectory from dir1 ).
+#mkdir -p dir1/dir2/dir3
```

Comment:
as mentioned in the question you need to create the directories under home directory because of ~ sign so you need to include it with your command like `mkdir -p ~/dir1/dir2/dir3` or `cd ~` then `mkdir -p dir1/dir2/dir3`